research/object_detection/object_detection_tutorial.ipynb

```diff
@@ -257,10 +257,13 @@
       "outputs": [],
       "source": [
         "# patch tf1 into `utils.ops`\n",
-        "utils_ops.tf = tf.compat.v1\n",
+        "utils_ops.tf = tf.compat.v2\n",
```

Comment:
This colab is explicitly installing TF2 at the beginning, so if you are making it TF1&2 compatible, you should add a comment there to let people know that they can install either 1.15 or 2.

Reply:
Changed the __Setup section__ as:

> __Important:__ If you're running on a local machine, be sure to follow the [installation instructions](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/installation.md). 
> After finishing local installation, you can skip Install section on notebook and start from [Imports](#Imports) section.
> 
> Install section includes only what's necessary to run in Colab.

And on Install part added a comment as asked

## Also about indentation
2 whitespaces as indentation were shown red as if there is a error on the code on local Jupyter-Notebook, so i changed indentations to 4 whitespaces for eyecandy
From:
![indent-red](https://user-images.githubusercontent.com/6182106/82948524-99188880-9faa-11ea-97b0-60f2e4e1c746.png)
to:
![indent4](https://user-images.githubusercontent.com/6182106/82948592-b8afb100-9faa-11ea-8934-14fa3ec17c95.png)